src/js/configHolder.js

```diff
@@ -0,0 +1,153 @@
+"use strict";
+var fluid = require("infusion");
+var process = require("process");
+
+// TODO: Revisit once we can use a package-relative path.
+// https://github.com/fluid-project/markdownlint-config-fluid/pull/2
+var baseMarkdownlintConfig = require("markdownlint-config-fluid");
+
+fluid.defaults("fluid.lintAll.configHolder", {
+    gradeNames: ["fluid.component"],
+    config: {
+        rootPath: process.cwd(),
+        "sources": {
+            "css": ["./*.css", "./src/**/*.css", "tests/**/*.css"],
+            "js": ["./src/**/*.js", "./tests/**/*.js", "./*.js"],
+            "json": ["./src/**/*.json", "./tests/**/*.json", "./*.json"],
+            "json5": ["./src/**/*.json5", "./tests/**/*.json5", "./*.json5"],
+            "md": ["./src/**/*.md", "./tests/**/*.md", "./*.md"],
+            "scss": ["./*.scss", "./src/**/*.scss", "tests/**/*.scss"]
+        },
+        "minimatchOptions": {
+            "dot": true,
+            "matchBase": true
+        },
+        "eslint": {
+            "minimatchOptions": "{that}.options.config.minimatchOptions",
+            "rootPath": "{that}.options.config.rootPath",
+            "enabled": true,
+            "js": {
+                "minimatchOptions": "{that}.options.config.minimatchOptions",
+                "rootPath": "{that}.options.config.rootPath",
+                "enabled": true,
+                "includes": "{that}.options.config.sources.js",
+                "excludes": [],
+                "options": {
+                    "resolvePluginsRelativeTo": "@expand:fluid.module.resolvePath(%fluid-lint-all)",
```

Comment:
Nice!

Reply:
There was a brief hair-pulling moment where it seemed like we were headed down the same road of randomly having to require dev dependencies downstream, I was very happy to find this option.  FYI, I found this in testing the use of this package via `npx`, i.e. the strategy also works in that little sandbox.